src/schema/rules/sidecars/pet.yaml

```diff
@@ -150,12 +150,16 @@ PETReconstruction:
       description_addendum: This partly matches the DICOM Tag (0054,1103) `Reconstruction Method`.
     ReconMethodParameterValues:
       level: required
+      level_addendum: |
+        Required if `ReconMethodParameterLabels` does not contain 'none', optional if
+        no labels exist.
```

Comment:
I think this needs to be reflected in the selectors- i.e., we need two mutually exclusive sets of selectors. One with `"none" in sidecar.ReconMethodParameterLabels` and one with `!("none" in sidecar.ReconMethodParameterLabels)`.

Reply:
Also, shouldn't the level_addendum be applied to `ReconMethodParameterUnits` as well?

Reply:
Fits with the logic, I've never seen them have units so I don't pay them much mind.

Reply:
>I think this needs to be reflected in the selectors- i.e., we need two mutually exclusive sets of selectors. One with "none" in sidecar.ReconMethodParameterLabels and one with !("none" in sidecar.ReconMethodParameterLabels).

Are you suggesting the below?

```
RuleOne:
  selectors: 
      - "none" in sidecar.ReconMethodParameterLabels
  fields:
    ReconMethodParameterUnits: optional
    ReconMethodParameterValues: optional

RuleTwo:
   selectors:
       - !("none" in sidecar.ReconMethodParameterLabels)
   fields:
     ReconMethodParameterUnits: required
     ReconMethodParameterValues: required
```

Do we really need mutual exclusion for this? It seems excessive when one rule covers both cases, or is it a technical limitation with the validator or schema tools?